### PR TITLE
Create gradle.yml

### PR DESCRIPTION
**Pull request by PGavrilenko**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Gradle
```
Comment (SantaMinus) [inline .github/workflows/gradle.yml]: Unneeded comment. Can start with `name:`

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
```
Comment (SantaMinus) [inline .github/workflows/gradle.yml]: Unneeded new line

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (PGavrilenko) [reply .github/workflows/gradle.yml]: Done

File: .github/workflows/gradle.yml (hunk 2 above)
Comment (PGavrilenko) [reply .github/workflows/gradle.yml]: Done

**Review by SantaMinus — APPROVED** [review]

**Comment by SantaMinus** [comment]
👍

**Pull request merged**